### PR TITLE
Use built-in wgsl type aliases instead of our own, remove zero/one wgsl constants

### PR DESCRIPTION
**Pull request by Wumpf**
### What

Stumbled over the existence of those while working on another thing.
Zero/one constants seem to be pointless given how easily the values are initialized - default ctor for all types is defined to be zero :)

_commit by commit_

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/4200) (if applicable)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4200)
- [Docs preview](https://rerun.io/preview/c329bdcdfc69278bbcc19a00113274087f24dabf/docs) 
- [Examples preview](https://rerun.io/preview/c329bdcdfc69278bbcc19a00113274087f24dabf/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: crates/re_renderer/shader/composite.wgsl [hunk 1]
```diff
@@ -27,7 +27,7 @@ fn main(in: FragmentInput) -> @location(0) vec4f {
     // but are about the location of the texel in the target texture.
     var color = textureSample(color_texture, nearest_sampler, in.texcoord).rgb;
     // TODO(andreas): Do something meaningful with values above 1
-    color = clamp(color, ZERO.xyz, ONE.xyz);
+    color = clamp(color, vec3f(), vec3f(1.0));
```
Comment (emilk) [inline crates/re_renderer/shader/composite.wgsl:30]: I'd rather be explicit here with `vec3f(0.0)`

**Review by emilk — APPROVED** [review]
LGTM if tested on mobile!

**Comment by Wumpf** [comment]
m-o-b-i-l-e.... oof. haven't touched that in ages 🤔
will give it a spin. extremely unlikely to affect it because it _should_ generate the same glsl code. gotta test that should ofc 😁

**Comment by Wumpf** [comment]
confirmed working on my Android on firefox & chrome

**Pull request merged**